code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -928,6 +928,24 @@
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/do_teleport, M, get_turf(M), 5, null, null, null, 'sound/effects/phasein.ogg'), 30)
 	..()
 
+/datum/reagent/brass
+	name = "Brass"
+	id = "brass"
+	description = "Brass is a alloy of Iron and Teslium. Popular among steampunk enthusiasts and cultists, who claim that it has divine properties."
+	reagent_state = SOLID
+	color = "#B5A642" // rgb: 181, 166, 66
+	taste_description = "gears"
+	var/initial = null
+	
+/datum/reagent/brass/on_mob_add(mob/living/carbon/human/M)
```

Comment:
what's the default argument here? within your PR I see one with an arg of carbon/human and another with just living.

The argument needs to be the most generic (probably /mob/living) and you set your human specific items within the proc using istype then blah/blah/human/H = M and so forth.

Reply:
wrong, argument can be anything for easy istypes

Reply:
that's why you must use istype(H), but you don't need to use istype(H,/mob/living/carbon/human)

Reply:
yeah I guess you could call that correct, togopal